src/core/Akka/Actor/ActorPath.cs

```diff
@@ -175,6 +175,31 @@ public IReadOnlyList<string> Elements
         }
 
         /// <summary>
+        /// INTERNAL API.
+        /// 
+        /// Used in Akka.Remote - when resolving deserialized local actor references
+        /// we need to be able to include the UID at the tail end of the elements.
+        /// 
+        /// It's implemented in this class because we don't have an ActorPathExtractor equivalent.
+        /// </summary>
+        public IReadOnlyList<string> ElementsWithUid
```

Comment:
Since we don't have the concept of an `ActorPathExtractor`, I bolted this onto `ActorPath` directly. Works the same as `ActorPath.Elements` except that it bolts the UID on as a Uri fragment to the very last element. Needed for making all of the Akka.Remote UID guarantees workable.